lib/helpers/ips/Ips.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SendGrid;
+class Ips
+{
+	private $sg;
+	
+	public function __construct($sg)
```

Comment:
Our code base is compatible with type hinting, let's type hint here because I think you're expecting `$sg` to be an instance of `SendGrid` object, e.g:

```php
public function __construct(SendGrid $sendGrid)
```